safenode/src/node/api.rs

```diff
@@ -129,20 +129,28 @@ impl Node {
             NetworkEvent::RequestReceived { req, channel } => {
                 self.handle_request(req, channel).await?
             }
-            NetworkEvent::PeerAdded => {
+            NetworkEvent::PeerAdded(peer) => {
```

Comment:
I think merge `PeerAdded` and `PeersAdded` will save a lot of traffic on `get_closest`

Reply:
Fixed!